website/docs/rbi.md

```diff
@@ -151,42 +155,79 @@ a method's arity, they can also declare a method's input and output types.
 
 Hand-written RBIs for gems can come from either:
 
-- [`sorbet-typed`], a central repository for sharing hand-written RBI files with
+- [`rbi-central`], a central repository for sharing hand-written RBI files with
```

Comment:
Should we add a note here explaining rbi-central vs sorbet-typed?

Reply:
Do you mean a deprecation warning or a link to this note in your blogpost?

Reply:
I was thinking maybe just put something here like

```markdown
> **Note**: With [`srb rbi` moving to maintenance mode](blog-post), [`rbi-central`] has shifted to being the main repository of community-sourced RBI files for open source gems, replacing [`sorbet-typed`].
```

Reply:
Done in f099f437d 👍